src/Type/ShipmentOrdersGet.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Collmex ShipmentOrdersGet Type
+ *
+ * @author    Marcus Jaschen <mail@marcusjaschen.de>
+ * @author    René Galle <renegalle.webdevelopment@gmail.com>
+ * @license   http://www.opensource.org/licenses/mit-license MIT License
+ * @link      https://github.com/mjaschen/collmex
+ */
```

Comment:
Please let's not add new file dock blocks. (I've removed the existing ones in #81.) Thanks!

Reply:
But the property stuff is to be kept right? :D

Reply:
Yes, the *class* doc block should stay. Only the (redundant) *file* doc block should go away.

Reply:
Got it^^ I will adjust it and push the commit as soon as I‘m on the pc again ;)